leptos/src/lib.rs

```diff
@@ -204,7 +204,7 @@ pub mod prelude {
 pub mod form;
 
 /// A standard way to wrap functions and closures to pass them to components.
-pub mod callback;
+pub use reactive_graph::callback;
```

Comment:
Just experimenting; can we trick the semver check by using the following?

```rust
pub use reactive_graph::callback as callback;
```

Reply:
The failed semver error is clear that `A publicly-visible module cannot be imported by its prior path` (leptos/src/callback.rs). A dummy module in `leptos/src/callback.rs` with:

```rust
pub use reactive_graph::callback::*;
```

and in `lib.rs`:

```rust
pub mod callback;
```

may satisfy the checks but I don't know if that would be a viable option!

Reply:
If that works, a TODO tag should be wrriten to remove the redundant module in next major release.